subgraphs/stakedao/src/common/constants.ts

```diff
@@ -0,0 +1,67 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+
+export namespace Network {
+  export const AVALANCHE = "AVALANCHE";

```

Comment:
Update these constants with the ones in the reference subgraph (coingecko slugs)